pkgs/development/python-modules/cbor/default.nix

```diff
@@ -1,23 +0,0 @@
-{ lib, buildPythonPackage, fetchPypi }:
-
-buildPythonPackage rec {
-  pname = "cbor";
-  version = "1.0.0";
-  format = "setuptools";
-
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1dmv163cnslyqccrybkxn0c9s1jk1mmafmgxv75iamnz5lk5l8hk";
-  };
-
-  # Tests are excluded from PyPI and four unit tests are also broken:
-  # https://github.com/brianolson/cbor_py/issues/6
-  doCheck = false;
-
-  meta = with lib; {
-    homepage = "https://bitbucket.org/bodhisnarkva/cbor";
```

Comment:
Ok. I was confused. But the current homepage is actually: https://github.com/brianolson/cbor_py/

Reply:
That's okay. I'm also confused :)

Would you mind creating another PR updating the homepage? However, the dead link is still marked as the homepage [on PyPI](https://pypi.org/project/cbor/).

Unfortunately, the library seems dead to me.

Reply:
https://github.com/NixOS/nixpkgs/pull/306281